telco-ran/configuration/README.md

```diff
@@ -0,0 +1,14 @@
+# Reference configuration
+
+## Structure
+This directory contains three components of the reference configuration
+ - The `source-crs` tree contains the CRs which form the foundation of
+   the RAN reference configuration.
+ - The `extra-manifests-builder` folder is used to create the extra-manifests in `source-crs/extra-manifest` directory.
+ - The `kube-compare-reference` tree contains the RAN reference configuration CRs with templating required for kube-compare tool.
+ - The `argocd` folder contains documentation and templates in `examples`
+ - The `examples/policygentemplates` tree has the reference manifests which define how CRs from the
```

Comment:
Maybe rephrase the end to `grouped into ordered policies which apply the reference configuration to one or more clusters.`